### PR TITLE
Fix translate link in moderation queue (bug 1141711)

### PR DESCRIPTION
**Pull request by eviljeff**

### CODE REVIEW

File: mkt/reviewers/views.py [hunk 1]
```diff
@@ -918,7 +918,7 @@ def _retrieve_translation(text, language):
 
 
 @waffle_switch('reviews-translate')
-@reviewer_required
+@permission_required([('Apps', 'ModerateReview')])
```
Comment (diox) [inline mkt/reviewers/views.py]: wouldn't `@reviewer_required(moderator=True)` work and be more consistent with the rest ?

Comment (eviljeff) [inline mkt/reviewers/views.py]: `` @reviewer_required(moderator=True) `` gives access to reviewers *or* moderators.  Translation is used in the moderation queue, which is inaccessible to users with only Apps:Review permission.

Comment (diox) [inline mkt/reviewers/views.py]: ah, ok

**Comment by diox** [comment]
r+

**Pull request merged**